### ISSUE TITLE
Can you share the face mask for the dataset

### ISSUE THREAD

**Issue by 07hyx06** [USTC3DV/NeRFBlendShape-code#15]
Hi, thanks for your excellent work!
I tried to obtain the face mask for each frame using the [face-parsing.PyTorch](https://github.com/zllrunning/face-parsing.PyTorch) repo as suggested in the paper, but the results are not as good as Fig 6, see the Obama example predicted directly from face-parsing.PyTorch:
![00020](https://user-images.githubusercontent.com/52267384/215686213-2eaf6b4a-9ab8-4f01-8d97-7a1f03821966.png)

Can you release the face mask or provide some insights on obtaining more accurate masks?

Thanks.

**Comment by XuanGhahahaha** [comment]
Hi, our segmented results are not more accurate than yours. Athough there will be a little noise and flicking in segmented GT face images, NeRF tends to synthesize a denoised and smooth result when training is finished. Just using these segmented images to supervise training is enough.

**Comment by 07hyx06** [comment]
Got it, thanks for your reply
